acceptance-tests/src/test/resources/uk/gov/di/test/features/Account_management_apis.feature

```diff
@@ -0,0 +1,15 @@
+@AccountManagement @build @staging
+  Feature: Account Management
+    Scenario: A user can reset their password after successful sign in
+      Given the user comes from the stub relying party with options: "scopes-account-management"
+      Then the user is taken to the "Create your GOV.UK One Login or sign in" page
+      When the user selects sign in
+      Then the user is taken to the "Enter your email" page
+      When user enters "TEST_USER_ACCOUNT_MANAGEMENT_EMAIL" email address
+      Then the user is taken to the "Enter your password" page
+      When the user enters their password
+      Then the user is taken to the "Check your phone" page
+      When the user enters the six digit security code from their phone
+      Then the user is returned to the service
+      When the user is calling the Account Management API with the access token
```

Comment:
maybe say that it's not actually the user who does this as they can't call the api directly, also say which endpoint is being called in the description